packages/create-nx-workspace/bin/create-nx-workspace.ts

```diff
@@ -442,27 +440,29 @@ function createApp(
     : ` --interactive=false`;
   const defaultBaseArg = defaultBase ? ` --defaultBase="${defaultBase}"` : ``;
 
-  console.log(
-    `new ${name} ${args} --preset="${preset}"${appNameArg}${styleArg}${nxCloudArg}${interactiveArg}${defaultBaseArg} --collection=@nrwl/workspace`
-  );
-  const executablePath = path.join(tmpDir, 'node_modules', '.bin', cli.command);
-  const collectionJsonPath = path.join(
-    tmpDir,
-    'node_modules',
-    '@nrwl',
-    'workspace',
-    'collection.json'
+  const packageExec = getPackageManagerExecuteCommand(packageManager);
+  const command = `new ${name} ${args} --preset="${preset}"${appNameArg}${styleArg}${nxCloudArg}${interactiveArg}${defaultBaseArg} --collection=@nrwl/workspace`;
+  console.log(command);
+
+  const collectionJsonPath = require.resolve(
+    '@nrwl/workspace/collection.json',
+    { paths: [tmpDir] }
   );
+
   execSync(
-    `"${executablePath}" new ${name} ${args} --preset="${preset}"${appNameArg}${styleArg}${nxCloudArg}${interactiveArg}${defaultBaseArg} --collection=${collectionJsonPath}`,
+    `${packageExec} tao ${command.replace(
+      '--collection=@nrwl/workspace',
+      `--collection=${collectionJsonPath}`
+    )} --cli=${cli.command} --root=${process.cwd()}`,
```

Comment:
I thought the point of having root is not to use `process.cwd()`, cause process.cwd is used when root isn't provided